docs/guides/introduction/about.md

```diff
@@ -16,8 +16,8 @@ These packages provide tools and architectures that allow you to quickly build s
 
 Daffodil is built with flexibility and scalability in mind. Here are a few features that we believe make Daffodil special:
 
-- **Drivers** [Drivers](/docs/guides/drivers.md) are Daffodil's lifeblood. The drivers enable you to easily integrate with a variety of ecommerce backends, payment systems, and other services, allowing you to extend functionality without disrupting the core framework.
-- **Design Components** - Building trust is key in ecommerce. With [Daffodil's component kit](/docs/design), streamline development and create a polished, professional store that customers will love.
-- **Performance**: With a focus on speed, Daffodil ensures a smooth and fast user experience for both customers and administrators, even as your store grows.
+- **Drivers**: [Drivers](/docs/guides/drivers.md) are Daffodil's lifeblood. The drivers enable you to easily integrate with a variety of ecommerce backends, payment systems, and other services, allowing you to extend functionality without disrupting the core framework.
+- **Design Components**: Building trust is key in ecommerce. With [Daffodil's component kit](/docs/design), you can streamline development and create a polished, professional store that customers will love.
+- **Performance**: Performance is paramount. Your store can only be performant if our code is fast, you can rely on us to prevent and fix performance issues as quickly as possible.
```

Comment:
Performance is paramount. Your store can only be performant if our code is fast. You can rely on us to prevent and fix performance issues as quickly as possible.